packages/sentry/src/sentry.ts

```diff
@@ -21,6 +22,38 @@ const sendErrorToSentry = (error: Reportable) => {
   return Sentry.captureException(errorToCapture);
 };
 
+type TReplacements = {
+  [name: keyof Event]: string;
+};
+
+const replaceEventValues = (
+  event: Event,
+  replacements: TReplacements
+): Event => {
+  for (const prop in event) {
+    if (event.hasOwnProperty(prop)) {
+      const hasPropReplacement = Object.keys(replacements).includes(prop);
+
+      switch (typeof event[prop]) {
+        case 'string':
+          if (hasPropReplacement) {
+            event[prop] = replacements[prop];
+          }
+
+          break;
+        case 'object':
```

Comment:
What about arrays?

Reply:
I'd say `[Redacted]`. Not like first element this, second element that.

Reply:
No I mean does the recursion work also for arrays or nested array objects?

Reply:
Yes, arrays are objects. I added a test for the case.